features/plugin-activate.feature

```diff
@@ -62,6 +62,27 @@ Feature: Activate WordPress plugins
       Plugin 'user-switching' activated.
       """
 
+  Scenario: Activating a plugin with no network wide option passes down correct types
```

Comment:
```suggestion
  @require-php-7
  Scenario: Activating a plugin with no network wide option passes down correct types
```

@raicem This should fix the errors on PHP 5.6

Scalar type hints are only available in PHP 7+

Reply:
Oh, thank you @swissspidy for the suggestion. I have applied this suggestion now.